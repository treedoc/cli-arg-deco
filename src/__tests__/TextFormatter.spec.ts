import TextFormatter from "../TextFormat";


const str1 = `ab\tcdef\tghijk
a\tbcdefg\th
a\tb\tcde\tfg\th
`
const str2 = `abcd
efg
hijk
lmn
`

describe('TextFormatter', () => {
  test('testAlignTabs', () => {
    expect(TextFormatter.alignTabs(str1)).toMatchSnapshot();
  });

  test('testIndent', () => {
    expect(TextFormatter.indent(str2, "  ")).toMatchSnapshot();
  })
});