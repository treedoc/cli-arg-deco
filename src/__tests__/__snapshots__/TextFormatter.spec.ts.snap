// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextFormatter testAlignTabs 1`] = `
"ab cdef   ghijk
a  bcdefg h
a  b      cde   fg h

"
`;

exports[`TextFormatter testIndent 1`] = `
"  abcd
  efg
  hijk
  lmn
  "
`;
